docker-compose.yml

```diff
@@ -11,7 +11,7 @@ services:
       - .:/source
     environment:
       - "NEO4J_ACCEPT_LICENSE_AGREEMENT=yes"
-      - "NEO4J_AUTH=neo4j/admin"
+      - "NEO4J_AUTH=none"
```

Comment:
Why did you change this ?  I prefer to keep the authentication because it's the most common case for neo4j

Reply:
So I was having some issues with travis and getting the tests to run, I will try again with this reverted.